ci-operator/config/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-main.yaml

```diff
@@ -2,15 +2,16 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.17
+    tag: rhel-8-release-golang-1.19-openshift-4.14
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
 - as: unit
-  commands: make test
+  commands: GOFLAGS= XDG_DATA_HOME=/tmp/.local/share XDG_CACHE_HOME=/tmp/.cache make
```

Comment:
Shouldn't we vendor dependencies (as a downstream patch) and remove the GOFLAGS override?

Reply:
Possibly, but since we're only using CAPO as a library in MAPO (we vendor in MAPO), is it important?